README.md

```diff
@@ -410,6 +412,37 @@ and **optional**.
   * Optional methods are ones this module does not call at all, but helps
     present uniform stores to users.
 
+Some methods (`createSession`, `load`, and `regenerate`) are already available from the `express-session` module. These methods are marked as "available from `express-session`" below. Instead of writing these methods yourself, it is recommended that you pass the `express-session` object to your store on initialization, and extend your store from `expressSession.Store`. See below for an example:
+
+In `app.js/server.js/index.js/etc.`:
+
+```
```

Comment:
Addressed in https://github.com/expressjs/session/pull/354/commits/9c0ea61fb2f842d54cbbbb535d2a0b5a9bae98f2